.github/workflows/update-oam-defs.yml

```diff
@@ -0,0 +1,104 @@
+    env:
+      NEW_APP_MESH_VERSION: ${{ needs.check_app_mesh_version.outputs.NEW_APP_MESH_VERSION }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+      - name: Get App Mesh manifests
+        run: |
+          mkdir ./app-mesh-crds
+          cd ./app-mesh-crds
+          curl --remote-name-all `curl -H "Accept: application/vnd.github.VERSION.raw" https://api.github.com/repos/aws/aws-app-mesh-controller-for-k8s/contents/config/crd/bases\?ref\=$NEW_APP_MESH_VERSION | jq 'map(.download_url) | .[]' | tr -d '"'`
+          cd ..
+          awk 'FNR==1 && NR>1 { printf("\n%s\n\n","---") } 1' app-mesh-crds/*.yaml > ~/app.yaml
+      - name: Build jsonschema util
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+          nvm install v14.15.3
+          nvm install-latest-npm
+          git clone https://github.com/layer5io/kubeopenapi-jsonschema util
+          cd util
+          npm i; npm i nexe -g
+          make linux
+      - name: Generate Workload definitions
+        run: |
+          dirPath=../templates/oam/workloads/workloads-$NEW_APP_MESH_VERSION
+          cd util
+          mkdir -p $dirPath
+          meshName=appmesh
+          smpMeshName=APP_MESH
+          template='{"apiVersion":"core.oam.dev/v1alpha1","kind":"WorkloadDefinition","metadata":{},"spec":{"definitionRef":{},"metadata":{"@type":"pattern.meshery.io/mesh/workload","meshVersion":"'$NEW_APP_MESH_VERSION'","meshName":"'$smpMeshName'","k8sAPIVersion":null,"k8sKind":""}}}'
+          crds=$(./kubeopenapi-jsonschema --location ~/app.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition")]' -o json --o-filter '$..["spec"]["names"]["kind"]' | jq '.[]')
+          for t in ${crds[@]}; do
+            nameUpper=`echo $t  | tr -d '"'`
+            nameLower=`echo $t  | tr -d '"' | tr '[:upper:]' '[:lower:]'`
+            definitionRef=$(printf %s.appmesh.meshery.layer5.io $nameLower)
+            apiVersion=$(./kubeopenapi-jsonschema --location ~/app.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..spec.versions[0]' --o-filter "$[].name" -o json | jq '.[]' | tr -d '"')
+            apiGroup=$(./kubeopenapi-jsonschema --location ~/app.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..spec' --o-filter "$[].group" -o json | jq '.[]' | tr -d '"')
+            ./kubeopenapi-jsonschema --location ~/app.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..openAPIV3Schema.properties.spec' --o-filter "$[]" -o json |\
+            jq '
+            .[]
+            | .version = "'$NEW_APP_MESH_VERSION'"
```

Comment:
version, object type and service mesh should be removed from here

Reply:
made the changes